src/components/LogsTab/LogsTools/index.tsx

```diff
@@ -52,7 +53,7 @@ class LogsTools extends React.PureComponent<Props, State> {
 
   public render(): React.ReactElement {
     const logsText = this.props.logs.join('\n');
-    const logsBlobUrl = URL.createObjectURL(new Blob([logsText], { type: 'text/x-log' }));
+    const logsBlobUrl = getBlobUrl ? getBlobUrl(logsText) : '';
```

Comment:
Why it's necessary to check if function `getBlobUrl` is imported?

Reply:
It is not necessary. I have removed it.

Reply:
It was needed first for testing.